public/app/features/admin/ldap/LdapDrawer.tsx

```diff
@@ -341,10 +341,10 @@ export const LdapDrawerComponent = ({
                     onChange={(v) => {
                       setValue(
                         `${serverConfig}.root_ca_cert_value`,
-                        v.filter(({ v }) => typeof v === 'string')?.map(({ v }) => v)
+                        v.filter((v) => typeof v.value === 'string').map(({ value }) => String(value))
                       );
                     }}
-                    value={watch(`${serverConfig}.root_ca_cert_value`)?.map((v) => ({
+                    value={watch(`${serverConfig}.root_ca_cert_value`, []).map((v) => ({
```

Comment:
The same thing with using `Controller` here, you'd not need to use watch for the value like this, instead it'd be provided via `register`.

Reply:
Changed as requested